src/main/res/values/strings.xml

```diff
@@ -1592,16 +1592,16 @@
     -->
     <string name="subscription_expired">Expired</string>
 
-    <!-- Translators "Last used" is a date, time or date+time attribute -->
+    <!-- Translators: "Last used" is a date, time or date+time attribute -->
     <string name="last_used_on">Last used</string>
     <string name="payment_method">Payment method</string>
     <string name="payment_method_cash">Cash</string>
     <string name="payment_method_credit_card">Credit card</string>
     <string name="payment_method_debit_card">Debit card</string>
     <string name="payment_method_transit_card">Transit card</string>
     <string name="payment_method_cheque">Cheque</string>
-    <!-- Translators: Free as in "gratis" (costing no money) -->
-    <string name="payment_method_free">Free</string>
+    <!-- Translators: As in free of charge (costing no money) -->
+    <string name="payment_method_free">Gratis</string>
```

Comment:
I'm not sure that this is an improvement. Not every user of English is a native speaker. Most of our English users most likely just don't have a translation for their language. Maybe "Free of charge"?

Reply:
The term _gratis_ is rarely used by native English speakers outside of [the free software community](https://www.gnu.org/philosophy/free-sw.en.html) and [particular British social classes who like dropping Latin phrases ~as a form of bullying~ to sound intelligent](https://medium.com/@garius/the-problem-with-the-romans-is-they-always-try-to-walk-it-in-65a961e59648).  It also risks confusion to those speaking English as a second language who don't _also_ know a Romance language.

Given the context necessary to understand it, I fear that this change risks alienating users.

The translation comment is there for languages that distinguish between "_free_ as in costing no money" and "_free_ as in liberty" (such as Romance languages).  If it merely had the word "free" without context, this would be risky to translate correctly.

I'm not sure what we'd actually gain out of adding more words here, given that the words "Payment method" is printed immediately before it in a list.

I figure that folks translating stuff are more likely to understand "gratis" quickly (either through their language skills or their association with the free software community), and at worst, it is explained for those who don't (without implying any passing of judgement). :)

Reply:
I was a bit quick there perhaps, a mix of thinking there was no context, and seeing something having to be explained.
"Payment method" is actually "none", how about "Payment" "Free"
"Free/gratis" maybe, but that is a smaller fish to fry.